pkg/services/accesscontrol/acimpl/service.go

```diff
@@ -702,7 +702,10 @@ func (s *Service) searchUserPermissions(ctx context.Context, orgID int64, search
 		permissions = s.actionResolver.ExpandActionSetsWithFilter(permissions, GetActionFilter(searchOptions))
 	}
 
-	key := accesscontrol.GetSearchPermissionCacheKey(&user.SignedInUser{UserID: userID, OrgID: orgID}, searchOptions)
+	key, err := accesscontrol.GetSearchPermissionCacheKey(s.log, &user.SignedInUser{UserID: userID, OrgID: orgID}, searchOptions)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I'm not sure if we should error out or just log and not set the cache if we can't compute the cache key.

Reply:
Yeah that's a good point. I think you have a good point that just logging will be a better experience for users.

Reply:
Fixed here: 4265aaa62e8b58d5bb3319b3f735d712b491cce6